uptane-standard.md

```diff
@@ -98,83 +98,128 @@ TODO
 
 # Detailed Design of Uptane
 
-## Repositories and Servers
+## Roles on repositories
 
-### Image Repository
+### The Root role {#root_role}
 
-### Director Repository
+### The Targets role {#targets_role}
 
-### Time Server
+ <!-- TODO sub-headings about delegations -->
+
+### The Snapshot role {#snapshot_role}
+
+### The Timestamp role {#timestamp_role}
+
+## Metadata abstract syntax
+
+### Common Metadata Structures and Formats
+
+### Root Metadata {#root_meta}
+
+### Targets Metadata {#targets_meta}
+
+#### Metadata about Images
 
-## Primary vs. Secondary Verification
+#### Metadata about Delegations {#delegations}
 
-## Roles
+### Snapshot Metadata {#snapshot_meta}
 
-### The Root Role
+### Timestamp Metadata {#timestamp_meta}
 
-### The Targets Role and Delegations
+## Server / repository implementation requirements
 
-#### Prioritized Delegations
+An Uptane implementation SHALL make the following services available to vehicles:
 
-#### Terminating Delegations
+* Image repository
+* Director repository
+* Time server
 
-#### Multi-Role Delegation
+### Image Repository
+
+The Image repository exists to allow an OEM and/or its suppliers to upload images and their associated metadata. It makes these images and their metadata available to vehicles. The Image repository is designed to be primarily controlled by human actors, and updated relatively infrequently.
```

Comment:
Is the Images repository really "primarily controlled by human actors"? It can be automatically managed just like the Director repository.

Reply:
I suppose that it might be better to explain that the image metadata uploaded is signed by keys that are kept offline, thus requiring human intervention. The description of how the metadata in the image repo is signed is still to be written, though...let's expand this description with a reference once that section is done.

Reply:
Ah, I see what you mean now. Yes, that plan sounds fine.